site/css/donate.scss

```diff
@@ -51,12 +51,8 @@
 
     .donation-options {
         display: flex;
-        & > * {
-            margin-right: 8px;
-            &:last-child {
-                margin-right: 0;
-            }
-        }
+        flex-wrap: wrap;
```

Comment:
this makes the custom amound field wrap on desktop

Reply:
Can you be more specific? It doesn't wrap for me on desktop in any of my browsers.

Reply:
Ah, I should have waiting for this before the merged. Forgot about it. I hope it's not a big issue since I wasn't able to reproduce it.

Reply:
Now I see that it wraps only on big screen. Fixing.

Reply:
Fixed in https://github.com/owid/owid-grapher/commit/bc08abb57.